components/tidb_query_vec_expr/src/impl_string.rs

```diff
@@ -492,6 +492,12 @@ pub fn to_base64(bs: &Option<Bytes>) -> Result<Option<Bytes>> {
     }
 }
 
+#[rpn_fn]
+#[inline]
+pub fn lower(arg: &Option<Bytes>) -> Result<Option<Bytes>> {
+    Ok(arg.as_ref().map(|b| b.to_vec()))
```

Comment:
The `lower` func in the normal version hasn't been separated into `lower` and `lower_utf8` yet. Would you like to post a PR to implement it :)? . You can refer to `upper` and `upper_utf8` in this PR #6280.

Reply:
OK, thanks. I want to try it.